infrastructure/infrastructure.py

```
@@ -0,0 +1,476 @@
+            except Exception as e:
+                print(f"Error during cleanup: {e}")
+
+def setup_instance_chain(instance: Instance) -> Instance:
+    """Set up an instance: create -> network -> setup script"""
+    instance = create_instance(instance)
+    instance = create_network(instance)
+    instance = run_setup_script(instance)
+    return instance
+
+def connect_local(manager: Instance):
+    TAILSCALE_TAILNET = os.getenv("TAILSCALE_TAILNET")
```

Ruff (0.13.0)

Local variable `TAILSCALE_TAILNET` is assigned to but never used

Remove assignment to unused variable `TAILSCALE_TAILNET`

(F841)